modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -16,93 +16,183 @@ All scopes in the endpoint map directly to permissions available in Anypoint Pla
+* Your app is owned by your organization; any organization administrator can control it.
+* A unique `clientID` is automatically assigned and can’t be modified. The `clientSecret` has no expiration date by default but can be modified.
 
-== Endpoint Scopes
+You should also determine the use-case on whether the app is internal to your company or is a third-party application that can be distributed outside of your organization. To create your app, follow these steps:
 
-Most of the scopes in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.well-known/openid-configuration[discovery endpoint] map directly to existing permissions in Anypoint Platform. Other available scopes are defined as follows:
+include::partial$include-nav-connected-apps.adoc[]
 
-* `full`: Full access to anything the user can do.
-* `read:full`: Read-only access to anything the user can read.
-* `openid`: Read-only access to the user's username and unique Anypoint ID.
-* `profile`: Read-only access to the user's Anypoint profile.
-* `email`: Read-only access to the user's email.
-* `offline_access`: Access to the user's data when they are not logged in.
+. In the *Owned Apps* section, click *Create App*.
+. In the *Create App* page, complete the following fields:
+.. Name +
+A Unique name for the application
++
+.. *Type* +
+The type of Connected App to be created. Based on the type selected, the required fields varies.
++
+.. *Grant types* +
+The grant types to be used with this app. +
+Note: Additional information may be requested based on the grant type you select. For more information on grant types, see xref:connected-apps-developers.adoc#grant-types-and-token-flow[Grant types and token flow].
++
+.. *Website URL* +
+Users can visit this URL to learn more about the app.
++
+.. *Redirect URIs* +
+Configure which URIs users should be directed to after authorization. Type the URL and click *Add* to add it to the list.
++
+.. *Scopes* +
+The level of permissions to be provided to the app. Click on the *Add Scopes* button to select the scopes. For more information, see xref:connected-apps-developers.adoc#endpoint-scopes[Endpoint Scopes].
++
+. Click *Save*
 
 [NOTE]
-Apps that use `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries using the scope correspond to the user who created the connected app.
+====
+The connected app ownership gets transferred to the root organization owner
+when the user who created the connected app:
 
-For more information on scopes and permissions in the Connected Apps feature, see the scopes documentation page in Access Management:
+* is deleted
+* is removed from the root organization
+* loses administrative privileges over the organization
+
+By clicking on the app name in Access Management > *Connected Apps* > *Owned Apps*, organization administrators can edit the
+owner of the connected app and transfer the ownership to other administrators.
```

Comment:
Organization administrators can edit the owner of the connected app and transfer the ownership to other administrators. To do so, click the app name in Access Management > *Connected Apps* > *Owned Apps* and make the appropriate edits.

Reply:
that is just me being picky ^